File: src/main/java/com/qtech/bubbles/command/HealCommand.java
package com.qtech.bubbles.command;

import com.qtech.bubbles.entity.player.PlayerEntity;

public class HealCommand implements CommandExecutor {
    @Override
    public boolean execute(PlayerEntity player, String[] args) {
        player.setDamageValue(player.getMaxDamageValue());
        return true;
    }
}

File: src/main/kotlin/com/qtech/bubbles/command/HealCommand.kt
package com.qtech.bubbles.command

import com.qtech.bubbles.common.command.CommandExecutor
import com.qtech.bubbles.entity.player.PlayerEntity

open class HealCommand : CommandExecutor {
    override fun execute(player: PlayerEntity, args: Array<String>): Boolean {
        player.damageValue = player.maxDamageValue
        return true
    }
}